src/tokens/saga.test.ts

```diff
@@ -252,9 +256,65 @@ describe('watchAccountFundedOrLiquidated', () => {
   })
 
   it('does not dispatch the account funded event for an account restore', async () => {
+    jest.mocked(getDynamicConfigParams).mockReturnValue({ showBalances: ['celo-alfajores'] })
     await expectSaga(watchAccountFundedOrLiquidated)
       .provide([
-        [select(lastKnownTokenBalancesSelector), dynamic(balances(null, new BigNumber(10)))],
+        [
+          select(lastKnownTokenBalancesSelector, [NetworkId['celo-alfajores']]),
+          dynamic(balances(null, new BigNumber(10))),
+        ],
+      ])
+      .dispatch({ type: 'TEST_ACTION_TYPE' })
+      .dispatch({ type: 'TEST_ACTION_TYPE' })
+      .run()
+
+    expect(ValoraAnalytics.track).toHaveBeenCalledTimes(0)
+  })
+
+  it('does not dispatch the account funded event when network ID added', async () => {
+    jest
+      .mocked(getDynamicConfigParams)
+      .mockReturnValueOnce({ showBalances: ['celo-alfajores'] })
+      .mockReturnValueOnce({ showBalances: ['celo-alfajores', 'ethereum-sepolia'] })
+    await expectSaga(watchAccountFundedOrLiquidated)
+      .provide([
+        [
+          select(lastKnownTokenBalancesSelector, [NetworkId['celo-alfajores']]),
+          dynamic(balances(new BigNumber(0), new BigNumber(0))),
```

Comment:
one static return value should be sufficient here right?

Reply:
yep, I was just following the pattern from the other tests cause I wasn't sure if it needed to be in this format but that makes sense. Just updated this and all other places where I did this.